src/app/cli/src/init/test_submit_to_archive.ml

```diff
@@ -529,8 +540,36 @@ let generate_all_transactions ~(precomputed_values : Precomputed_values.t)
          ~message:"Sender's account not found in ledger by location"
     |> Account.nonce
   in
+  let account_state_tbl = Account_id.Table.create () in
+  let genesis_accounts = Mina_ledger.Ledger.to_list_sequential genesis_ledger in
+  List.iter genesis_accounts ~f:(fun account ->
+      let account_id = Account.identifier account in
+      let role =
+        if Account_id.equal account_id signer_account_id then `Fee_payer
+        else `Ordinary_participant
+      in
+      Account_id.Table.set account_state_tbl ~key:account_id
+        ~data:(account, role) ) ;
+
+  (* TODO: Figure out how to create zkApp accounts for max cost generation *)
+  let account_state_tbl =
+    if max_cost then failwith "TODO: Figure out how to create zkApp accounts"
+    else account_state_tbl
+  in
+
+  let vk =
```

Comment:
I'm confused all the tweaks here. The blocks are still not cryptographically valid. so what's the point instead of feeding garbage into archive? It's a test anyway and we don't verify anything AFAIK on archive's end.